.github/workflows/python.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: Pylint
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10", "3.11", "3.12"]
```

Comment:
Should this include 3.6 and 3.9 for el8 and el9 clients?

Reply:
I will put here whatever people want I have no strong opinion on that. See previous comment from @bastian-src https://github.com/theforeman/smart_proxy_salt/pull/100#discussion_r1869519277

Reply:
Ah, interesting, so even on those old platforms it wants py310